client/source/DelphiLint.Plugin.pas

```diff
@@ -303,7 +304,12 @@ constructor TPluginCore.Create(Owner: TComponent; IDEServices: IIDEServices);
 //______________________________________________________________________________________________________________________
 
 procedure TPluginCore.OnRegister;
+var
+  NewPath: string;
 begin
+  NewPath := TPath.Combine(LintContext.Settings.SettingsDirectory, 'bin') + ';' + GetEnvironmentVariable('PATH');
+  SetEnvironmentVariable(PWideChar('PATH'), PWideChar(NewPath));
```

Comment:
With this change, I think we should also adjust the install script to not change the PATH anymore.

Reply:
I was considering it. I want to make sure that this works correctly in all cases before removing it from the install script, since if it doesn't work all the time then there's nothing the user can do. It doesn't do any active harm being paranoid and adding it to the PATH.

I'll probably improve the install script soon to have it download the default SonarDelphi version - someone suggested it on DelphiPraxis and I thought it was a good idea - so I could probably bundle the removal up with those changes. What do you think?

Reply:
You're right that it doesn't do any active *harm*, but I don't see how we'll ever be any **more** confident that we can remove the PATH manipulation from the installer.

Reply:
Sure.

Reply:
I've removed the path configuration from the install script now.